src/System.Private.CoreLib/src/System/Buffer.cs

```diff
@@ -2,6 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if AMD64 || (BIT32 && !ARM)
```

Comment:
We don't set `BIT32` anywhere. If you think we should set it, you'll need to add it to dir.props at the root and validate it kicks in on the TFS side as well (there's a separate place where we set stuff like `BIT64` under `ndp\FxCore\dir.settings.targets` - it might be redundant or might not, I didn't really look at how those two builds are tangled together).